main.py

```diff
@@ -70,6 +72,38 @@ def report(ctx, output):
     exporter.generate(project, 'report.html')
     pass
 
+@cli.command()
+@click.option('--version-file', default='version.csv', help='CSV Version file')
+@click.option('--overwrite', default=False, help='Overwrite database')
+@click.pass_context
+def import_csv(ctx, version_file, overwrite):
+    """Import CSV file into tables"""
+    # Overwrite option
+    if overwrite:
+        session.query(Version).delete()
+        click.echo("Clearning version table")
+
+    # Read Version CSV file
+    versions = list(csv.DictReader(open(version_file, 'r')))
+    
+    for version in versions:
+        # Instanciation with Version model
+        newVersion = Version(project_id=version['project_id'],
+        name=version["name"], 
+        tag=version["tag"], 
+        start_date=datetime.strptime(version["start_date"], '%Y-%m-%d %H:%M:%S.%f'), 
+        end_date=datetime.strptime(version["end_date"], '%Y-%m-%d %H:%M:%S.%f'), 
+        bugs=version["bugs"], 
+        changes=version["changes"], 
+        avg_team_xp=version["avg_team_xp"], 
+        bug_velocity=version["bug_velocity"])
+
+        # Save on database
+        session.add(newVersion)
```

Comment:
You should catch the exception and display a meaningful message for the end user. Especially if a key constraint is violated.